tests/modules/nf-core/mageck/test/main.nf

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { MAGECK_TEST } from '../../../../../modules/nf-core/mageck/test/main.nf'
```

Comment:
You're only using the module once per workflow, so you only need to import it once.